### PR TITLE
Allow modification of menu keybindings

### PR DESCRIPTION
**Pull request by Leandros**
My keyboard layout has different keys on 'j' & 'k'. This allows changing the defaults.

### CODE REVIEW

File: lib/nerdtree/menu_controller.vim [hunk 1]
```diff
@@ -71,9 +71,9 @@ endfunction
 "change the selection (if appropriate) and return 1 if the user has made
 "their choice, 0 otherwise
 function! s:MenuController._handleKeypress(key)
-    if a:key == 'j'
+    if a:key == 'j' || a:key == g:NERDTreeMenuDown
```
Comment (PhilRunninger) [inline lib/nerdtree/menu_controller.vim]: Change this to
```vim
if a:key == g:NERDTreeMenuDown
```
and initialize the variable in this section: https://github.com/scrooloose/nerdtree/blob/master/plugin/NERD_tree.vim#L116-L150

File: lib/nerdtree/menu_controller.vim [hunk 2]
```diff
@@ -71,9 +71,9 @@ endfunction
...
         call self._cursorDown()
-    elseif a:key == 'k'
+    elseif a:key == 'k' || a:key == g:NERDTreeMenuUp
```
Comment (PhilRunninger) [inline lib/nerdtree/menu_controller.vim]: Same change here.

**Comment by Leandros** [comment]
Reasonable. Will add the changes later today.

**Comment by Leandros** [comment]
Incorporated your suggestions. Although, I didn't update the `j/k` text, since by default you'd still use `j` and `k` for navigation. The variables are documented in NERDTree.txt.

**Comment by PhilRunninger** [comment]
That text needs to change. The default is `j/k`, but if you override them, that needs to be reflected in the text.
```vim
    echo "NERDTree Menu. Use " . g:NERDTreeMenuDown . "/" . g:NERDTreeMenuUp . "/enter and the shortcuts indicated"
```

**Comment by Leandros** [comment]
Ahh, I didn't think of that. Makes sense.

**Comment by Leandros** [comment]
Fixed.

**Comment by PhilRunninger** [comment]
Excellent! Thanks for the contribution, @Leandros.

**Comment by Leandros** [comment]
Thank you for NERDTree!

**Pull request merged**